planning/behavior_path_planner/include/behavior_path_planner/scene_module/pull_over/pull_over_module.hpp

```diff
@@ -209,6 +215,8 @@ class PullOverModule : public SceneModuleInterface
   void updateOccupancyGrid();
   void researchGoal();
   void resetStatus();
+  bool checkCollision(const Pose & pose) const;
+  bool checkCollision(const PathWithLaneId & path) const;
```

Comment:
I am not sure, but if it is a template function, I understand that they are the same name.
But if not, I was wondering if the names should not be the same.

Reply:
It's just overload. changed functions name in https://github.com/autowarefoundation/autoware.universe/pull/1777/commits/ad499414df941db5dd86900a164f3613df76b7b9

Reply:
Sorry... simple overload is not problem at all according to the google style guide :bow: 
https://google.github.io/styleguide/cppguide.html#Operator_Overloading

So the previous one is enough but the new one is of cause perfect for me.